zebra-state/src/service/finalized_state/zebra_db/block.rs

```diff
@@ -388,6 +389,11 @@ impl ZebraDb {
 
         Ok(finalized.verified.hash)
     }
+
+    /// Writes the given batch to the database.
+    pub(crate) fn write_batch(&self, batch: DiskWriteBatch) -> Result<(), rocksdb::Error> {
+        self.db.write(batch)
+    }
```

Comment:
We're exposing direct write access to the database via arbitrary batches here. It would be better if we prevented that, but I can't think of a better solution.

Reply:
Would it help to move methods here so this type defines all of the possible db operations that Zebra may make?

Reply:
I don't know.

Reply:
Optional: open an issue with the problem? It may be worth fixing ahead of more format upgrades after the subtree indexes, or before stabilizing the zebra-state crate.